frontend/src/components/ExperimentsTable.jsx

```diff
@@ -44,7 +43,7 @@ const ExperimentsTable = (props) => {
 
   const resultList = resultKeys.map((resultId) => results[resultId]);
 
-  const logs = xAxisKeys.map((logKey) => ({
+  const logs = experiments.logKeys.map((logKey) => ({
```

Comment:
We can use `show` option of Columns component in react-table.
https://github.com/react-tools/react-table#columns

I think this is a better approach to manage columns visibility.

Reply:
Thank you! I try to use `show` option.